doozer/doozerlib/backend/konflux_olm_bundler.py

```
@@ -865,6 +878,32 @@ def _find_external_digest_images(
+            operator_manifests_dir: Path to the operator's manifests directory.
+
+        Returns:
+            Set of image names defined in external-images config, or empty set if none.
+        """
+        art_yaml_path = operator_manifests_dir / 'art.yaml'
+        if not art_yaml_path.is_file():
+            return set()
+        try:
+            with open(art_yaml_path, 'r', encoding='utf-8') as f:
+                art_yaml_data = yaml.safe_load(f.read())
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)